bin/blat-filter-fusion.py

```diff
@@ -48,17 +48,14 @@
 
 
 def myfilter(txt):
-    r = False
     lin = txt.rstrip('\r\n').split('\t')
-    if ( lin and
+    return ( lin and
         (float(lin[0])/float(lin[10])) >= 0.70 and  # number of matches / length of query sequence
         lin[17] == '2' and  # blockCount
         lin[6] == '1' and   # number inserts in target = tNumInserts
         (lin[4] == '0' or (lin[4] == '1' and int(lin[5]) <= 3)) and  # number inserts in query
         min(map(int,lin[18][:-1].split(','))) > 9  # blockSizes
         ):
```

Comment:
One comment regarding the generate_banned.py. MALA1-KLK3 has been added but I guess that you meant MALAT1-KLK3. Right?